packages/table/src/interactions/selectable.tsx

```diff
@@ -93,6 +101,27 @@ export class DragSelectable extends React.Component<IDragSelectableProps, {}> {
             return false;
         }
 
+        const regionCardinality = Regions.getRegionCardinality(region);
+        let focusCellCoordinates: IFocusedCellCoordinates = null;
+
+        if (regionCardinality === RegionCardinality.FULL_TABLE) {
+            focusCellCoordinates = { col: 0, row: 0 };
+        }
+
+        if (regionCardinality === RegionCardinality.FULL_COLUMNS) {
+            focusCellCoordinates = { col: region.cols[0], row: 0 };
```

Comment:
Just want to make sure I understand: when we've selected a region with `FULL_COLUMNS`, is the top-left cell _in the body_ returned as the focused cell, or is the top-left _column-header cell_ returned? Same question for the `FULL_ROWS` case below.